Sources/BraintreeApplePay/BTApplePayClient.swift

```diff
@@ -114,14 +119,4 @@ import BraintreeCore
             "paymentNetwork": token.paymentMethod.network
         ]
     }
-
-    func completionHandler(
-        onMainThreadWithPaymentRequest paymentRequest: PKPaymentRequest?,
-        error: Error?,
-        completion: @escaping (PKPaymentRequest?, Error?) -> Void
-    ) {
-        DispatchQueue.main.async {
-            completion(paymentRequest, error)
-        }
```

Comment:
Doing some testing, we don't seem to need to call just this completion from the main queue. Ran a series of tests on device and simulator and `paymentRequest` is not throwing warnings. Should be good to remove this and call the completion the same way in both methods.